.github/workflows/generate-pinval.yaml

```diff
@@ -3,26 +3,27 @@ name: generate-pinval
 on:
   workflow_dispatch:
     inputs:
-      run_id:
+      year:
         type: string
         description: >
-          Run ID: The model run whose comps should be used
-        required: true
-        default: "2025-02-11-charming-eric"
+          Assessment year: Use values and comps from the final model for this year.
+          Leave blank if you are supplying a run ID instead
+        required: false
+        default: ""
 
-      triad_name:
+      run_id:
         type: string
         description: >
-          Triad name: Generate reports for every PIN in this triad. Leave blank if you are
-          supplying PINs instead. Options: city, north, south
+          Run ID: The model run whose comps should be used. Leave blank if you are supplying
+          an assessment year instead
         required: false
         default: ""
 
       pins:
         type: string
         description: >
-          PINs: One or more comma‑separated Cook County PINs. Leave blank if you are
-          supplying a triad instead
+          PINs: One or more space‑separated Cook County PINs. Leave blank to
```

Comment:
While I was deep in refactoring the PIN-parsing logic in this workflow, I decided to change the delimiter from commas to spaces, since that matches the interface for other similar workflows (e.g. [`build-and-test-dbt`](https://github.com/ccao-data/data-architecture/blob/e5bf4680d86e92443103974a04edb4492dcd1f5a/.github/workflows/build_and_test_dbt.yaml#L13)) and also more closely matches the interface of the underlying `generate_pinval.py` script.